tests/core/test_reader.py

```diff
@@ -1114,3 +1115,74 @@ def test_prostar_reader():
 
     mesh = reader.read()
     assert all([mesh.n_points, mesh.n_cells])
+
+
+def test_grdecl_reader():
+    def read(content, include_content, **kwargs):
+        path = Path(tempfile.mkdtemp())
```

Comment:
Suggest to use [the tmp_path fixture](https://docs.pytest.org/en/stable/how-to/tmp_path.html).
```suggestion
def test_grdecl_reader(tmp_path):
    def read(content, include_content, **kwargs):
        path = tmp_path
```